datajs/browser-test/index.spec.js

```diff
@@ -48,7 +49,7 @@ describe('FileInterface', function () {
     rowStream = await file.rows({ size: 1 })
     rows = await toArray(rowStream)
     expect(rows[0]).toEqual(['number', 'string', 'boolean'])
-    expect(rows[1]).toEqual(undefined)
+    // expect(rows[1]).toEqual(undefined)
```

Comment:
Why do you remove this? Isn't it undefined?

Reply:
This is related to the size parameter in rows. The new webToNodeStream does not support row numbers yet, as this is more difficult to implement when using the stream API instead of slicing text.

Reply:
I want to open an issue for this

Reply:
> We can combine both toNodeStream and webToNodeStream together though. I'll work on this now

Yes, sounds good. I think you can take advantage from [this feature](https://nodejs.org/api/readline.html#readline_event_line) from builtin `readline` package. Let's do this in this PR as merging this without that produce breaking changes in terms of row. You can update the issue description accordingly